examples/minimal/packages/client-react-external-wallet/src/mud/setupNetwork.ts

```diff
@@ -30,6 +32,13 @@ export async function setupNetwork() {
   const onWrite = (write: ContractWrite) =>
     write$.next({ id: `${write.id}:${nextWriteId++}`, request: write.request, result: write.result });
 
+  const burnerWalletClient = createWalletClient({
+    ...clientOptions,
+    account: privateKeyToAccount(getBurnerPrivateKey("mud:example:burnerWallet")),
+  })
+    .extend(burnerActions)
+    .extend(setupObserverActions(onWrite));
```

Comment:
I think creating a burner wallet at the very beginning like this is the easiest approach to implement. However, generating burner wallets for users without external wallets seems unnecessary, making it less than ideal. If I decide to create burner wallets only when needed, I'm considering introducing a new React context and state for this purpose, which would allow easy access to burner wallets for underlying components.